google/import_google_project_services_test.go

```diff
@@ -0,0 +1,51 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccGoogleProjectServices_importBasic(t *testing.T) {
+	t.Parallel()
+
+	resourceName := "google_project_services.acceptance"
+	projectId := "terraform-" + acctest.RandString(10)
+	conf := testAccGoogleProjectServices_import(projectId, org, pname)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: conf,
+			},
+			resource.TestStep{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateId:     projectId,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccGoogleProjectServices_import(projectId, orgId, projectName string) string {
```

Comment:
Since this is so similar to testAccGoogleProjectAssociateServicesBasic from `resource_google_project_services_test.go`, what if you just called that function instead of having a new one here?

Reply:
Good catch, done.